.travis.yml

```diff
@@ -26,11 +18,13 @@ matrix:
       perl: '5.28'
 notifications:
   email:
+    if: repo = "tokuhirom/OrePAN2"
     on_failure: always
     on_success: always
     recipients:
       - olaf@wundersolutions.com
 perl:
+  - '5.10.0'
```

Comment:
I don't think this is doing what was intended: https://travis-ci.org/tokuhirom/OrePAN2/jobs/622959535#L145-L239

Maybe we need to switch to `trusty`.  Some of the other Perls are also reverting to 5.22.1

Reply:
Yes, I see what you mean.

However, this build, https://travis-ci.org/davel/OrePAN2/jobs/620867872 shows a failure exactly like I would expect for 5.10.0, yet 5.22.1 is reported. Compare with any of http://matrix.cpantesters.org/?dist=OrePAN2%200.47;perl=5.10.0;reports=1

Could the helper scripts somehow be reporting the wrong Perl yet running the correct one?

Reply:
That's interesting.  I don't have a good answer for that, but I don't see anything in the log file that suggests it's 5.10 that is running the code.

Reply:
Yes, it looks like Travis is asking the wrong Perl to report it's version. In a branch I added perl 5.8 to the list, and it complains about `use feature`, as I'd expect for 5.8. It does however report 5.22.1 in the prologue. See https://travis-ci.org/davel/OrePAN2/jobs/623314181

Reply:
Ah, I was wrong!  https://travis-ci.org/davel/OrePAN2/jobs/623314181#L283-L298  https://github.com/travis-perl/helpers#notes  I've been burned by this before.  :(  Thanks for pointing that out.  It's really confusing.